test/parallel/test-cli-syntax.js

```diff
@@ -0,0 +1,55 @@
+var assert = require('assert');
+var spawnSync = require('child_process').spawnSync;
+var path = require('path');
+
+var common = require('../common');
+
+var node = process.argv[0];
+
+// test both sets of arguments that check syntax
+var syntax_args = [
+  ['-c'],
+  ['--check']
+];
+
+// test good syntax with and without shebang
+[
+  'syntax/good_syntax.js',
+  'syntax/good_syntax_shebang.js'
+].forEach(function(file) {
+  file = path.join(common.fixturesDir, file);
+
+  // loop each possible option, `-c` or `--check`
+  syntax_args.forEach(function(args) {
+    var _args = args.concat([file]);
+    console.log('calling %s %s', node, _args.join(' '));
+    var c = spawnSync(node, _args, {encoding: 'utf8'});
+
+    // no output should be produced
+    assert.equal(c.stdout, '', 'stdout produced');
+    assert.equal(c.stderr, '', 'stderr produced');
+    assert.equal(c.status, 0, 'code == ' + c.status);
+    console.log('ok');
+  });
+});
+
+// test bad syntax with and without shebang
+[
+  'syntax/bad_syntax.js',
+  'syntax/bad_syntax_shebang.js'
+].forEach(function(file) {
+  file = path.join(common.fixturesDir, file);
+
+  // loop each possible option, `-c` or `--check`
+  syntax_args.forEach(function(args) {
+    var _args = args.concat([file]);
+    console.log('calling %s %s', node, _args.join(' '));
+    var c = spawnSync(node, _args, {encoding: 'utf8'});
+
+    // no output should be produced
+    assert.equal(c.stdout, '', 'stdout produced');
+    assert(c.stderr, 'stderr not produced');
+    assert.equal(c.status, 1, 'code == ' + c.status);
+    console.log('ok');
```

Comment:
Can you remove the console.log statements in this file?  Rule of thumb is that tests should be (mostly) quiet on success.